easybuild/easyconfigs/j/Jansson/Jansson-2.13.1-GCC-11.2.0.eb

```diff
@@ -0,0 +1,40 @@
+# Contribution from Imperial College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'Jansson'
+version = "2.13.1"
+
+homepage = 'https://www.digip.org/jansson/'
+description = """Jansson is a C library for encoding, decoding and manipulating JSON data.
+ Its main features and design principles are:
+ * Simple and intuitive API and data model
+ * Comprehensive documentation
+ * No dependencies on other libraries
+ * Full Unicode support (UTF-8)
+ * Extensive test suite"""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/akheron/jansson/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['f22901582138e3203959c9257cf83eba9929ac41d7be4a42557213a22ebcc7a0']
+
+# For configure, the ld.gold linker does not know anything about --default-symver and thus crashes
+# So we simnply use the bfd linker
+# preconfigopts = 'autoreconf -i && CC="$CC -fuse-ld=bfd" '
+# This is not required with CMake
+
+configopts = '-DJANSSON_BUILD_SHARED_LIBS=ON '
+
+builddependencies = [('CMake', '3.22.1')]
+
+sanity_check_paths = {
+    'files': ['lib/libjansson.%s' % SHLIB_EXT],
+    'dirs': ['include'],
+}
+
+runtest = 'check'
+
+moduleclass = 'devel'
```

Comment:
@sassy-crick maybe `data` rather than `devel`?

Reply:
Sounds good to me. I read library so I jumped to `devel`. :-)